docs/advanced/specification.md

```diff
@@ -2,7 +2,7 @@
 
 MDX includes a [specification][spec] to define the syntax and transpilation.
 This can be leveraged by code formatters, linters, and implementations in other languages created by the community.
-It's based on the [remark][remark]/[unist][unist] ecosystem to ensure robust parsing and the ability to leverage plugins from within your MDX.
+It's based on the [remark][]/[unified][] ecosystem to ensure robust parsing and the ability to leverage plugins from within your MDX.
```

Comment:
The text said "unist" but linked to https://unifiedjs.github.io/. Should it read "unist" and link to https://github.com/syntax-tree/unist, or is this change correct?

Reply:
That change is indeed correct 🙏